im.nheko.Nheko.yaml

```diff
@@ -0,0 +1,247 @@
+id: im.nheko.Nheko
+command: im.nheko.Nheko
+runtime: org.kde.Platform
+runtime-version: '6.6'
```

Comment:
6.7 is the current verison

Reply:
We haven't finished testing on 6.7 yet, which is why I went with 6.6 for now, which is what we tested for the last few months. I plan to bump it to 6.7 in a few weeks after we had some time to test it. Would that be okay?

Reply:
Yea